packages/fast-components-react-base/src/breadcrumb/breadcrumb.schema.ts

```diff
@@ -1,30 +1,30 @@
-export default {
-    $schema: "http://json-schema.org/schema#",
-    title: "Breadcrumb",
-    description: "A breadcrumb component's schema definition.",
-    type: "object",
-    id: "@microsoft/fast-components-react-base/breadcrumb",
-    formPluginId: "@microsoft/fast-components-react-base/breadcrumb",
-    properties: {
-        label: {
-            title: "label",
-            type: "string",
-        },
-    },
-    reactProperties: {
-        children: {
-            title: "Children",
-            type: "children",
-            formPluginId: "@microsoft/fast-components-react-base/breadcrumb/children",
-            defaults: ["text"],
-        },
-        separator: {
-            title: "Separator",
-            type: "children",
-            formPluginId:
-                "@microsoft/fast-components-react-base/badbreadcrumbge/separator",
-            pluginId: "@microsoft/fast-components-react-base/breadcrumb/separator",
-            defaults: ["text"],
-        },
-    },
-};
+export default {
```

Comment:
any idea what changed in this file? I'm not sure I see what the difference is.

Reply:
Most likely prettier?

Reply:
looks like it's more likely a line endings issue as the last edit of these files may have come from a user who has a different setting than you. I've submitted an issue to correct this by overriding any users config with a config set by this repository: https://github.com/microsoft/fast-dna/issues/2914